docs/design/002-airdrop.md

```diff
@@ -0,0 +1,100 @@
+# Design Doc 002: Airdrop Module
+
+## Status
+
+Draft
+
+## Abstract
+
+For the Ojo Network's Airdrop, we intend to require that users are only able to claim the other half of their tokens by staking their entire first half with a validator.
+
+## Context
+
+We want to incentivize airdrop recipients to stake their tokens. An effective way to do this is by rewarding users with the other half of their airdrop allocation if they stake to a validator.
+
+We also need to ensure that we're able to create vesting accounts for the initial airdrop amount, and controlling how long afterwards it will take to unlock. An issue here is that vesting accounts in cosmos can only be created for accounts which do not yet exist.
```

Comment:
What is the vesting period going to be? I assume the time required to unstake is not enough.

Reply:
Vesting periods for genesis accounts haven't been finalized yet, but for umee's genesis this was a few years